src/index.js

```diff
@@ -35,19 +35,14 @@ function init( {
 				if ( popup.loading ) {
 					if ( data ) {
 						if ( data.type === 'standard' ) {
+							popup.lang = lang
+							popup.title = title
+
 							popup.show(
 								renderPreview( lang, data, isTouch ),
 								target,
 								pointerPosition
 							)
-
-							popup.lang = lang
-							popup.title = title
-							const expanded = root.querySelector( '.wikipediapreview.expanded.mobile' )
-
-							if ( expanded && popup.expand ) {
```

Comment:
@stephanebisson @medied do you remember why we need this here?

in `event.js`'s `onShow` event, we already have the function called `onExpand()`

Reply:
I don't remember... it seems redundant. It's OK to remove but we have to test the hell out of it.

Reply:
I think it's for the `requestPageMedia` and subsequent `getGalleryRow` for when the preview text is short enough to render with gallery row... but yea this can probably be improved

<img width="545" alt="image" src="https://user-images.githubusercontent.com/4752599/93889712-893e5600-fcb7-11ea-9916-6fe99534bfaf.png">

Reply:
> I think it's for the `requestPageMedia` and subsequent `getGalleryRow` for when the preview text is short enough to render with gallery row... but yea this can probably be improved

Got it, and there is already a check during `onShow` event, that should be enough.

Reply:
oh wait, i can see the reason why now @medied, let me rework on this.